### ISSUE TITLE
Update UseCases.md to document different input arguments for RunUseCase

### ISSUE THREAD

**Issue by sheryjoe** [SCIInstitute/ShapeWorks#538]

**Comment by jadie1** [comment]
I added this to UseCases.md in the org-doc branch for [PR #541 ](https://github.com/SCIInstitute/ShapeWorks/pull/541)

**Comment by AtefehKashani** [comment]
This kooks good to me, there are two tiny modification for this documentation.
1- In --use_subsample: it is mentioned that the samples are selected using k-mean clustering. The new sampledata function is using spectral clustering based on the distance matrix. In my opinion, it is enough to say it used clustering. 
2- In --start_with_prepped_data: It worth mentioning that since the optimization uses distance transforms, what we mean by prep_data, is the distance transform of shapes.

**Comment by jadie1** [comment]
Thanks @AtefehKashani did you make these changes in the org-doc branch or would you like me to add them?

**Comment by AtefehKashani** [comment]
@jadie1 No, I didn't. If you agree with this changes, you can go ahead and make the changes. Thanks

**Comment by jadie1** [comment]
@AtefehKashani Okay I added these changes. If you think it looks good now we can close this issue.

**Comment by AtefehKashani** [comment]
Looks good to me, Thanks @jadie1
